website/source/api/secret/identity/group-alias.html.md

```diff
@@ -74,7 +74,7 @@ This endpoint is used to update an existing group alias.
 - `name` `(string: entity-<UUID>)` – Name of the group alias.
 
 - `mount_accessor` `(string: "")` – Mount accessor to which this alias belongs
-  toMount accessor to which this alias belongs to.
+  to.
```

Comment:
Ditto.

Reply:
Alternatively, remove the second `to` in both sentences.  Either `to` is valid, just not both.

Reply:
Ty. Updated